.github/workflows/spelling.yml

```diff
@@ -0,0 +1,22 @@
+name: Spellcheck
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
```

Comment:
Add the below parameters to accelerate the step
```
with:
        fetch-depth: 1
```

Reply:
Default is already 1 https://github.com/actions/checkout#usage